signer-service/src/api/services/sep10.service.js

```diff
@@ -0,0 +1,76 @@
+const { Keypair } = require('stellar-sdk');
+const { TransactionBuilder, Networks } = require('stellar-sdk');
+const { fetchTomlValues } = require('../helpers/anchors');
+
+const { TOKEN_CONFIG } = require('../../constants/tokenConfig');
+const { SEP10_MASTER_SECRET } = require('../../constants/constants');
+
+const NETWORK_PASSPHRASE = Networks.PUBLIC;
+
+exports.signSep10Challenge = async (challengeXDR) => {
```

Comment:
This endpoint will receive the challenge, check the operations to ensure that only Anclap is valid, and that the to-be-authorized account is our master account.

It is of course not the safest.